Server/BackEndMonitoringServer/src/CLogicalDiskStatus.cpp

```diff
@@ -0,0 +1,83 @@
+#include "stdafx.h"
+
+#include "CLogicalDiskStatus.h"
+
+CLogicalDiskStatus* CLogicalDiskStatus::FactoryLogicalDiskStatus(
+	const std::string& disk_name,
+	EMemoryCountType count_type)
+{
+	CLogicalDiskStatus* p_created_object = new CLogicalDiskStatus(
+		disk_name, count_type);
+
+	if (!p_created_object->TryUpdateCurrentStatus())
```

Comment:
Memory leak in case of error!!!

Reply:
Fixed in [commit](https://github.com/ITA-Dnipro/BackEndMonitoring/commit/28775d4e4e211a6985cecfb05174a510800a77d4)